### ISSUE TITLE
error reading a frame

### ISSUE THREAD

**Issue by davidenitti** [Rolf-Hempel/PlanetarySystemStacker#41]
I have an error reading a frame of a video, but it results aborting the entire process.
sometimes the video has the last frame corrupted because I have an out of space.
even when I cut a video with ffmpeg I get a similar error (error in reading video frame, index 30) in all the chunks I make.
Another case is when a frame was not stabilized.
is it possible to just skip the not readable/not stabilized frame? this would be much better.
thank you very much for the help!

**Comment by Rolf-Hempel** [comment]
Sorry for the late reply. I was thinking about this issue for a while. Unfortunately, the problem is more complicated than one might expect. When PSS opens a video, it reads the number of frames from the file metadata. At that point it has not read a single frame yet. The "frames" class in PSS is programmed such that frames can be read in random order (not neccessarily in consecutive order). If I then dropped a bad frame, the number of frames (and the indices of the good frames after the bad one) would change in the process. That would break the entire workflow.

Therefore, I decided not to change the program behaviour in this point. Sorry for that!
